org.bigtester.ate.core/src/main/java/org/bigtester/ate/model/caserunner/CaseRunner.java

```diff
@@ -237,7 +226,14 @@ public void initializeTestCase(TestParameters testParams) {
 		String[] configFiles = {testParams.getTestFilename()};
 		context = new FileSystemXmlApplicationContext(configFiles, testParams.getTestProject().getAppCtx());
 		IMyWebDriver myWebD = (IMyWebDriver) GlobalUtils
-				.findMyWebDriver(context);
+				.findMyWebDriver(context);		
+		WebDriver webD = myWebD.getWebDriverInstance();
+		try {
+			myWebD.getMultiWindowsHandler().refreshWindowsList(webD,false);
+		} catch (BrowserUnexpectedException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```

Comment:
handle printStackTrace,  this.retryRefreshWindows(getDriver(), false);

Reply:
Finished